src/bundle/Resources/config/services/components/search.yml

```diff
@@ -0,0 +1,10 @@
+services:
+    search.udw.subtree:
+        parent: EzSystems\EzPlatformAdminUi\Component\ScriptComponent
+        autowire: true
+        autoconfigure: false
+        public: false
+        arguments:
+            $src: '/bundles/ezplatformadminui/js/scripts/udw/select.location.js'
```

Comment:
Why do you need this add by the component? It's was created to add JS/CSS from another bundle, not from the same. I think you can add this easily in Twig.